### DISCUSSION TITLE
引入ep后,内存飙升,tsserver加载很久

### DISCUSSION THREAD

**Discussion by ghost** [element-plus/element-plus#17525] [Q&A]
vscode禁用所有插件(包括vue),使用vite新建vue ts项目,引入ep,内存疯涨到2G,新建.ts文件,内存再飙到3G,在新版vscode下,创建.ts,.vue文件,tsserver都要loading一次,时间很长.

然后我对其它一些vue3组件库(naiveui,antdv,tdv,arcov,varlet)进行了测试,内存维持在七八百上下,ts server加载时间较短

https://github.com/user-attachments/assets/cec2b624-6604-483c-9d46-da7dc72fc1e6

**Comment by warmthsea** [comment]
> vscode禁用所有插件(包括vue),使用vite新建vue ts项目,引入ep,内存疯涨到2G,新建.ts文件,内存再飙到3G,在新版vscode下,创建.ts,.vue文件,tsserver都要loading一次,时间很长.
> 
> 
> 
> 然后我对其它一些vue3组件库(naiveui,antdv,tdv,arcov,varlet)进行了测试,内存维持在七八百上下,ts server加载时间较短
> 
> 
> 
> https://github.com/user-attachments/assets/cec2b624-6604-483c-9d46-da7dc72fc1e6
> 
> 
> 
> 

如果不引入 ts 类型文件呢？

**Reply by warmthsea** [reply]
https://github.com/element-plus/element-plus/pull/16111

看看这个

**Reply by ghost** [reply]
试了下这个https://github.com/element-plus/element-plus/issues/16575#issuecomment-2067583895
,内存少了一半,loading加载快了一点,感觉还是很吃力
